### DISCUSSION TITLE
Switching to or from a scene with a NDI source makes latest OBS Studio crash (31.0.0) when tally is enabled

### DISCUSSION THREAD

**Discussion by EnkaHel** [DistroAV/DistroAV#1160] [Q&A - Support]
Greetings, 

DistroAV version: 6.0.0
NDI SDK version: WIN64 12:45:30 May 6 2024 6.0.1.0
OBS version : 31.0.0

When I switch to or from a scene with  NDI Source, OBS goes "Not Responding" and crashes the record and/or stream. 

So far, I was not able to reproduce this random behaviour with Program & Preview tally unchecked (meaning, disabled)

obs-log is not very helpful on the matter, since the program is "Not Responding" and not crashes abruptly

Regards,

**Comment by Trouffman** [comment]
if you are using HDR, this was detected and correct. Will be part of the 6.1 release (coming soon (tm) )
